src/components/Friend/WriteGroup.js

```diff
@@ -0,0 +1,319 @@
+import React, { useState } from "react";
+import {
+  Alert,
+  Image,
+  Modal,
+  Platform,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View
+} from "react-native";
+import * as ImagePicker from "expo-image-picker";
+import axios from "axios";
+
+import { firebase } from "../../firebase/Config";
+import { createGroup, editGroupDetails } from "../../firebase/HandleGroup";
+import { writeGroupType } from "../../constants/Group";
+
+const WriteGroup = ({ type, currentState, navigation }) => {
+  const [groupInfo, setGroupInfo] = useState(currentState);
+  const [modalVisible, setModalVisible] = useState(false);
```

Comment:
I think this entire component is extremely similar to the editProfileScreen, so I think this can be converted into a reuseable component instead?